### ISSUE TITLE
How many epochs was the model pre-trained for?

### ISSUE THREAD

**Issue by wangyuanbiubiubiu** [microsoft/VITRA#27]
I continued training the model on the vitra-1M dataset based on human_pretrain.json, and tried learning rates of 1e-4 and 1e-5 for the action branch respectively. However, the performance of the fine-tuned model was consistently worse than that of the original weights.

**Comment by YuDeng** [comment]
The released checkpoint is trained for 1–2 epochs. In our current experiments, further fine-tuning does not lead to additional performance gains. We believe that training for more steps would become beneficial when leveraging more diverse data.

**Comment by wangyuanbiubiubiu** [comment]
> The released checkpoint is trained for 1–2 epochs. In our current experiments, further fine-tuning does not lead to additional performance gains. We believe that training for more steps would become beneficial when leveraging more diverse data.

Thank you for your response. Additionally, I noticed that the learning rate for pre-training is constant. Have you tried using a cosine-based learning rate schedule? Would it lead to better results?

**Comment by YuDeng** [comment]
We haven’t experimented with learning rate schedules beyond a constant learning rate so far. It’s a good suggestion and could potentially improve performance.
